symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve, exp, sign)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+
+# Description
+## The barometric formula determines the dependence of the pressure or density of a gas on the height in the gravity field.
+
+## Law: p = p_0 * exp((-g * m * (h - h_0)) / (R * T))
+## Where:
+## p is the gas pressure in the layer located at height h
+## p_0 is the gas pressure at the initial height
+## g is acceleration of free fall
+## m is atomic weight of the gas
+## h is final height
+## h_0 is initial height
+## R is ideal gas constant
+## T is temperature
+
+## Conditions
+## Gas is ideal
+## The gas is in a uniform gravity field
+
+final_pressure = Symbol("final_pressure", units.pressure)
+initial_pressure = Symbol("initial_pressure", units.pressure)
+atomic_weight = Symbol("atomic_weight", units.mass / units.amount_of_substance)
+final_height = Symbol("final_height", units.length)
+initial_height = Symbol("initial_height", units.length)
+temperature = Symbol("temperature", units.temperature)
+
+law = Eq(final_pressure, initial_pressure * exp(units.acceleration_due_to_gravity * atomic_weight *
```

Comment:
Where is minus on gravity acceleration?

Reply:
When I put a minus in the law, it gives the wrong result, the same as if I count without a minus in the calculator, I did not understand why

Reply:
Because you have input parameters: initial_height, final_height. But in your test you pass: final_height, initial_height.